### PR TITLE
magento/merchdocs#1130: Describe product editing flow for Requisition List

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

Fix pull request fixes https://github.com/magento/merchdocs/issues/1130

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-dashboard-requisition-lists-manage.html

whatsnew
Updated [Managing Requisition Lists](https://docs.magento.com/user-guide/customers/account-dashboard-requisition-lists-manage.html) to describe product editing flow for Requisition Lists.

### CODE REVIEW

File: src/customers/account-dashboard-requisition-lists-manage.md [hunk 1]
```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
...
+
+Product in the Requisition List may be edit if:
+
+1. Product has **all options set** (in case you have a [configured product]({% link catalog/product-create-configurable.md %}) in the Requisition List).
+
+1. Product is **added to this Requisition List**.
+
+1. Product is [a simple product with options]({% link catalog/settings-advanced-custom-options.md %})
+
+1. Product's type is allowed to edit.
```
Comment (atwixfirster) [inline src/customers/account-dashboard-requisition-lists-manage.md:87]: Option is correct regarding to (core logic)[https://github.com/magento/partners-magento2b2b/blob/934c1ebff9f2a3c7389aadd2b30cffe89f33d684/app/code/Magento/RequisitionList/etc/di.xml#L32].

File: src/customers/account-dashboard-requisition-lists-manage.md [hunk 2]
```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
...
+1. Change any options that are necessary.
+
+1. Click **Update Requisition List** button.
+
+   ![]({% link images/images-b2b/update-requisition-list.png %}){: .zoom}
+   _Update Requisition List_
+
+Product in the Requisition List may be edit if:
+
+1. Product has **all options set** (in case you have a [configured product]({% link catalog/product-create-configurable.md %}) in the Requisition List).
```
Comment (atwixfirster) [inline src/customers/account-dashboard-requisition-lists-manage.md:81]: Option is correct regarding to (core logic)[https://github.com/magento/partners-magento2b2b/blob/934c1ebff9f2a3c7389aadd2b30cffe89f33d684/app/code/Magento/RequisitionList/Block/Requisition/View/Item.php#L360].

File: src/customers/account-dashboard-requisition-lists-manage.md [hunk 3]
```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
...
+
+1. Product is **added to this Requisition List**.
```
Comment (atwixfirster) [inline src/customers/account-dashboard-requisition-lists-manage.md:83]: Option is correct regarding to [core logic](https://github.com/magento/partners-magento2b2b/blob/934c1ebff9f2a3c7389aadd2b30cffe89f33d684/app/code/Magento/RequisitionList/Block/Requisition/View/Item.php#L363).

File: src/customers/account-dashboard-requisition-lists-manage.md [hunk 4]
```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
...
+
+1. Product is [a simple product with options]({% link catalog/settings-advanced-custom-options.md %})
```
Comment (atwixfirster) [inline src/customers/account-dashboard-requisition-lists-manage.md:85]: Option is correct regarding to [core logic](https://github.com/magento/partners-magento2b2b/blob/934c1ebff9f2a3c7389aadd2b30cffe89f33d684/app/code/Magento/RequisitionList/Model/Checker/ProductChangesAvailability.php#L46).

**Comment by atwixfirster** [comment]
Thank you for your contribution, @EvheniiaKolkhiieva !

From the technical side all points exist in a code base.

Thank you!

CC: @leslietilling

File: src/customers/account-dashboard-requisition-lists-manage.md [hunk 5]
```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
    ![]({% link images/images-b2b/requisition-list-print.png %}){: .zoom}
    _Print Requisition List_
 
+### Edit product options
+
+To edit a product options in the list do the following:
```
Comment (leslietilling) [inline src/customers/account-dashboard-requisition-lists-manage.md:68]:
```suggestion
To edit a product options in the list, do the following:
```

File: src/customers/account-dashboard-requisition-lists-manage.md [hunk 6]
```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
...
+
+1. Go to the product page via the **‘Pencil’** button. (![]({% link images/images/btn-edit.png %})).
```
Comment (leslietilling) [inline src/customers/account-dashboard-requisition-lists-manage.md:70]:
```suggestion
1.  Click the _Pencil_  (![]({% link images/images/btn-edit.png %})) icon to open the product page.
```

File: src/customers/account-dashboard-requisition-lists-manage.md [hunk 7]
```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
...
+
+1. Change any options that are necessary.
+
+1. Click **Update Requisition List** button.
```
Comment (leslietilling) [inline src/customers/account-dashboard-requisition-lists-manage.md:74]:
```suggestion
1. Click **Update Requisition List**.
```

File: src/customers/account-dashboard-requisition-lists-manage.md [hunk 8]
```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
...
+
+   ![]({% link images/images-b2b/update-requisition-list.png %}){: .zoom}
+   _Update Requisition List_
+
+Product in the Requisition List may be edit if:
```
Comment (leslietilling) [inline src/customers/account-dashboard-requisition-lists-manage.md:79]:
```suggestion
A product in the Requisition List may be edited if:
```

File: src/customers/account-dashboard-requisition-lists-manage.md (hunk 2 above)
Comment (leslietilling) [inline src/customers/account-dashboard-requisition-lists-manage.md:81]:
```suggestion
1. Product has **all options set** (when it is a [configured product]({% link catalog/product-create-configurable.md %}) in the Requisition List).
```

File: src/customers/account-dashboard-requisition-lists-manage.md (hunk 1 above)
Comment (leslietilling) [inline src/customers/account-dashboard-requisition-lists-manage.md:87]:
```suggestion
1. Edit is allowed for the product type.
```

**Review by leslietilling — COMMENTED** [review]
Some edits for grammar and style

**Comment by EvheniiaKolkhiieva** [comment]
Hi, @leslietilling,
Thank you for your reviewing!

I made some changes to the request, could you please check?

Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1131`.

**Comment by atwixfirster** [comment]
@dobooth , could you please add the next labels to this PR:
- `Partner: Atwix` 
- `partners-contribution`

Thank you!

**Pull request merged**